### PR TITLE
index defect + alpinejs attributes

### PR DESCRIPTION
**Pull request by hmbemba**
1. added a few attribute for alpine js

2. fixed index defect
```nim
import dekao

proc test: string = render:
  tdiv:
    p:
      say "Hello, world from test!"

let body = render:
  tdiv:
    say test()

echo body
```

### CODE REVIEW

File: src/dekao.nim [hunk 1]
```diff
@@ -310,6 +315,23 @@ proc attrHeight*(value: string) = attr "height", value
 proc hidden*(value: string) = attr "hidden", value
 proc high*(value: string) = attr "high", value
 proc href*(value: string) = attr "href", value
+
+proc `x-data`*(value: string) = attr "x-data", value
+proc `x-show`*(value: string) = attr "x-show", value
+proc `@click`*(value: string) = attr "@click", value
+proc `x-trans-es`*(value: string) = attr "x-transition:enter-start", value
+
+proc `asset-id`*(value: string) = attr "asset-id", value
```
Comment (ajusa) [inline src/dekao.nim]: These don't seem alpine related, could you remove them?

**Comment by ajusa** [comment]
Thank you for the PR!

Could you reduce this PR to just the index defect fixes? I'm glad you got the alpine attributes working, however I _personally_ don't use it, so I'd prefer not to include it here. You can always declare a module called `alpine` similar to the one called `htmx` in your own project and import it that way.

**Comment by hmbemba** [comment]
Got it just removed them!

**Review by ajusa — APPROVED** [review]
Thanks!

**Pull request merged**